### ISSUE TITLE
CPU version of tensorflow

### ISSUE THREAD

**Issue by chmille3** [yaserkl/RLSeq2Seq#14]
Does one need the gpu version of tensor flow? Can one just use the cpu version? Thanks!

**Comment by yaserkl** [comment]
No, soft placement in tensorflow will take care of cases where you don't have GPU. But, these models require GPU for training or the whole training process will take a long time on CPU.
